README.md

```diff
@@ -1,3 +1,25 @@
+---
+page_type: sample
+languages:
+- azdeveloper
+- csharp
+- powershell
+- shell
+- html
+- bicep
+products:
+- azure
+- azure-application-insights
+- azure-app-service
+- azure-sql-database
+- azure-front-door
+- enterprise-app-patterns
+urlFragment: reliable-web-app-pattern-dotnet
+name: Reliable web app pattern for .NET
+description: The Reliable Web App Pattern is a set of objectives to help your web application converge on the cloud. This repo contains a reference implementation of a reliable web application for .NET.
```

Comment:
This repo contains a reference implementation of a reliable web application hosted in a secure hub-spoke networking configuration for .NET.